values.yaml

```diff
@@ -3,7 +3,7 @@
 # Declare variables to be passed into your templates.
 
 # -- zabbix components (server, agent, web frontend, ...) image tag to use. Overwritten by zabbixserver.image.tag etc.
-zabbix_image_tag: ubuntu-6.2.0
+zabbix_image_tag: ubuntu-6.0.7
```

Comment:
This should be left commented by default, as now the tag defaults to AppVersion, right?

Reply:
Make sense, @sa-ChristianAnton! I will comment.

Reply:
@sa-ChristianAnton using the .Chart.appVersion don't works for Zabbix tags:
https://hub.docker.com/r/zabbix/zabbix-server-mysql/tags?page=1&name=6.0.7

In this case, the name of tag has distroname + version number.

The .Chart.appVersion not support distroname. I will remove the .Chart.appVersion of templates. Do you agree?